sky/cli.py

```diff
@@ -3467,9 +3473,37 @@ def storage_delete(names: List[str], all: bool):  # pylint: disable=redefined-bu
         \b
         # Delete all storage objects.
         sky storage delete -a
+        \b
+        # Skip confirmation prompt.
+        sky storage delete -y
```

Comment:
```suggestion
        sky storage delete imagenet -y
```
We need to provide a storage name to delete. Using a same storage as the example given above.